src/pages/choose-renewable-energy/index.js

```diff
@@ -65,16 +65,17 @@ const ChooseRenewablePage = () => (
               href="#actions"
               title="See how you can take action"
             >
-              <button
+              <a
                 className={`${ButtonStyles.btnSimple} ${ButtonStyles.btnMd} ${
                   ButtonStyles.btnBanner
                 } ${ButtonStyles.btnMobile}`}
+                aria-label={"Learn what you can do"}
               >
                 Learn what you can do
-              </button>
+              </a>
             </a>
             <div className={ChoiceStyles.bannerCtaSecondary}>
-              <a href="#how" title="How it happens">
+              <a href="/choose-renewable-energy/#how" title="How it happens">
```

Comment:
we don't want to reload the page, or scroll to the top here. This should scroll to the "How energy usage.." section. See how it is on live:

![Screencast 2019-06-05 at 9 29 25 pm](https://user-images.githubusercontent.com/26777813/58988307-4f52f300-87d9-11e9-8849-7a758a772a16.gif)